incubator/scylladb/templates/statefulset.yaml

```diff
@@ -84,27 +84,20 @@ spec:
           initialDelaySeconds: 15
           timeoutSeconds: 5
         volumeMounts:
-          - name: scylla-conf-yaml
+          - name: scylla-files
             mountPath: /etc/scylla/scylla.yaml
-          - name: scylla-ready-probe
+            subPath: scylla.yaml
+          - name: scylla-files
             mountPath: /opt/ready-probe.sh
+            subPath: ready-probe.sh
           {{- if .Values.persistence.size }}
           - name: datadir
             mountPath: /var/lib/scylla
           {{- end }}
       volumes:
-        - name: scylla-ready-probe
+        - name: scylla-files
           configMap:
             name: {{ template "scylladb.fullname" . }}
-            items:
```

Comment:
We do want this part, however; we want both scylla.yaml and ready-probe.sh, both of these referenced above...

Reply:
This is from my pod spec

```
    Mounts:
      /etc/scylla/scylla.yaml from scylla-files (rw)
      /opt/ready-probe.sh from scylla-files (rw)
Volumes:
  scylla-files:
    Type:      ConfigMap (a volume populated by a ConfigMap)
    Name:      user-likes-scylladb
    Optional:  false
```

Reply:
Oops, my bad, you're right. Thank you!